### PR TITLE
Nuclear counts loading

### LINKED ISSUE #646: Skip reading in nuclear segmentation labels when running generate_cell_table if nuclear_counts = False
Right now the function generate_cell_table inside of marker_quantification.py will read expect to read in the nuclear segmentation labels even if the input nuclear_counts = False. It does not seem to use this boolean variable until later in the call to create_marker_count_matrices. If nuclear_counts = False there is no reason to read in the nuclear labels.

### PR DESCRIPTION
**Pull request by camisowers**
**If you haven't already, please read through our [contributing guidelines](https://ark-analysis.readthedocs.io/en/latest/_rtd/contributing.html) before opening your PR**

**What is the purpose of this PR?**

Closes #646. 

**How did you implement your changes**

Update logic to only load nuclear image data if `nuclear_counts=True`.

### CODE REVIEW

File: ark/segmentation/marker_quantification.py [hunk 1]
```diff
@@ -519,18 +519,23 @@ def generate_cell_table(segmentation_dir, tiff_dir, img_sub_folder="TIFs",
...
+                                                               files=nuclear_files,
+                                                               xr_dim_name='compartments',
+                                                               xr_channel_names=['nuclear'],
+                                                               trim_suffix='_feature_1')
+            compartments = ['whole_cell', 'nuclear']
+            segmentation_labels = np.concatenate((current_labels_cell.values,
+                                                  current_labels_nuc.values), axis=-1)
+        else:
+            compartments = ['whole_cell']
+            segmentation_labels = current_labels_cell.values
```
Comment (alex-l-kong) [inline ark/segmentation/marker_quantification.py:533]: The `else` clause can be condensed to something like this:

```
compartments = ['whole_cell']
segmentation_labels = current_labels_cell.values

if nuclear_counts:
    current_labels_nuc = load_utils.load_imgs_from_dir(data_dir=segmentation_dir,
                                                       files=nuclear_files,
                                                       xr_dim_name='compartments',
                                                       xr_channel_names=['nuclear'],
                                                       trim_suffix='_feature_1')
    compartments.append('nuclear')
    segmentation_labels = np.concatenate((segmentation_labels,
                                          current_labels_nuc.values)), axis=-1)
```

**Review by alex-l-kong — COMMENTED** [review]
Just one minor code comment.

**Review by ngreenwald — CHANGES_REQUESTED** [review]
Looks good. Will you test out the segmentation notebook, deleting the `feature_1` tiffs that are generated by the deepcell call, to make sure there aren't any other hidden places where the nuclear images are assumed to exist

**Comment by camisowers** [comment]
> Looks good. Will you test out the segmentation notebook, deleting the `feature_1` tiffs that are generated by the deepcell call, to make sure there aren't any other hidden places where the nuclear images are assumed to exist

The last few code sections generating and saving the cell table ran without issue!

**Review by ngreenwald — APPROVED** [review]
Cool, good stuff

**Pull request merged**